packages/hagrid/hagrid/cli.py

```diff
@@ -3127,7 +3132,9 @@ def create_land_cmd(verb: GrammarVerb, kwargs: TypeDict[str, Any]) -> str:
         target = verb.get_named_term_grammar("node_name").input
         if target == "all":
             # subprocess.call("docker rm `docker ps -aq` --force", shell=True) # nosec
-            return "docker rm `docker ps -aq` --force"
+            if 'prune-vol' in kwargs:
```

Comment:
Really Appreciate the PR.

Currently , in hagrid, we have commands to do the same

like

```python
hagrid clean volumes < --- which prunes all the volumes
hagrid clean images   <---  which prunes all the images
hagrid clean containers < --- which lands all containers
```

You could search for `def clean` in cli.py of hagrid, which has these commands.

Reply:
I just realized the usecase of this PR.Ignore the above comment.

To able to prune volumes during land command, we need to add the --prune-vol option in `def land` instead of def launch which would make it appear in land command

Reply:
I have fixed it now